examples/stable_diffusion/faster_stable_diffusion_docker.yaml

```diff
@@ -0,0 +1,25 @@
+#SkyPilot YAML to run stable diffusion web tool on 1 V100 GPU.
+
+resources:
+  cloud: gcp
+  accelerators: V100:1
+
+file_mounts:
+  /faster_stable_diffusion: .
+
+setup: |
+  sudo rm -r stable-diffusion-webui-docker
+  git clone https://github.com/AbdBarho/stable-diffusion-webui-docker.git
```

Comment:
Just curious, if we are using the docker image from the GCR, why do we still need to git clone the original repo for the webui. We can store the code in the image, right?
We need to have some internal instructions for how to create that image from the original repo on GCR as well.

Reply:
I think we still need to clone the original repo on the instance, since we need to use and modify its files (docker-compose.yml and downloading the appropriate model weights). I was not sure if this was possible if the actual repo was inside the image, as this would involve modifying the image. Hence, I just kept it separate from the image. This also keeps the image size much smaller, as the weights itself is 4 GB, so that would potentially make the entire image >8 GB.

I will add instructions on how to create the image and upload to GCR.